CPT331.Web/Actions/ExportDataActionResult.cs

```diff
@@ -0,0 +1,37 @@
+﻿#region Using References
+
+using System;
+using System.Web.Mvc;
+
+using CPT331.Data;
+
+#endregion
+
+namespace CPT331.Web.Actions
+{
+	public class ExportDataActionResult : ActionResult
```

Comment:
We can now return a new type of action result from our controllers - this prompts the user to download a CSV.